game_helpers.h

```diff
@@ -0,0 +1,4 @@
+#ifndef GAME_HELPERS_H
+#define GAME_HELPERS_H
+
+#endif // GAME_HELPERS_H
```

Comment:
Mmm this seems like an empty file, maybe you can delete it. But we are always in need of game helpers.

Reply:
Yeah, this code was already there, Tianjian has not added it, hence I consider this resolved.